src/synthesizer/line_ratios.py

```diff
@@ -0,0 +1,116 @@
+"""
+A module holding useful line ratios (e.g. R23) and diagrams (pairs of ratios),
+e.g. BPT-NII.
+
+Line ids and specifically the wavelength part here are defined using the cloudy
+standard, i.e. using vacuum wavelengths at <200nm and air wavelengths at
+>200nm.
+
+There is a parallel module that vacuum
+"""
+
+# shorthand for common lines
+
+# standard names
+Ha = "H 1 6562.80A"
```

Comment:
I'm really confused as to why this is now a sub module with global attributes. I much preferred the previous implementation of a dedicated class. Alternatively, all of this information should just be defined on a line collection.

Reply:
Fair. I preferred the previous setup as users could then add their own ratios, and then make the most of all the downstream calculating and plotting functionality, whereas the new format is fixed

Reply:
I'm not sure it could be modified previously, the class had no initialiser so there's no interface to get an instance of it and then modify it.

Reply:
It was just a namespace, exactly the same as it is now just without the class keyword.

Reply:
Indeed, there isn't much difference between the previous implementations.

If we really wanted people to be able to add their own line ratios etc. we could add the line_ratio module as an argument to LineCollection. HOWEVER, methods already exist on LineCollection to quickly measure arbitrary ratios or diagrams by providing lists of lines. I've just improved this to make it easier so now you can do:

`lines.get_ratio('R23')` or `lines.get_ratio([line1, line2])`

Reply:
That's a nice solution. My impression was that the line ratios were a static thing?

I'm pretty sure (but haven't run a test) that if we wanted the user to be able to add to the line ratios in either the old container or the submodule structures we'd end up in one of those situations where we always get a copy and modify the original every time. That could lead to some really opaque bugs down the line. Better to have the line collection methods for arbitrary things.